src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const prevValue = new Set();
+  const copyRobot = {};
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
+      const currentValue = robot[key];
+
+      if (prevValue.has(currentValue)) {
```

Comment:
Better check if copyRobot hasOwnProperty with currentValue.
prevValue is actually redundant